projects/packages/my-jetpack/src/products/class-module-product.php

```diff
@@ -131,4 +133,64 @@ public static function deactivate() {
 		return Jetpack::deactivate_module( static::$module_name );
 	}
 
+	/**
+	 * Get Products that support this product
+	 *
+	 * @return array
+	 */
+	public static function get_supporting_products() {
```

Comment:
Could we use another function name like `get_containing_plans` to avoid getting confused with [get_supported_products](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/my-jetpack/src/products/class-product.php#L329)?

Reply:
What additional use cases does `get_supporting_products` cover when `get_supported_products` already exists?

Reply:
It's so hard to pick a right name 😅 

The existing `get_supported_products` is to get the product supported by the current product. For example: https://github.com/Automattic/jetpack/pull/31368/files#diff-96a3f7c8a02c743fc1ef8f8382674d897195688c05c45ef1d90b7422d0be465fR178

The `get_supporting_products` function I added is on the other hand is to get what products support the current product if it makes sense.

Reply:
> Could we use another function name like get_containing_plans to avoid getting confused with get_supported_products?

I guess I'll probably just go with `get_products`. What do you think? @dognose24